src/App.scss

```diff
@@ -0,0 +1,94 @@
+body {
+  min-height: 100vh;
+  background-color: #e9e6f2;
+  padding: 20px;
+  color: #4d457b;
+  font-family: "Roboto", sans-serif;
+}
+
+h2 {
+  font-size: 2rem;
+  text-align: center;
+  margin-bottom: 15px;
+}
+
+h3 {
+  text-align: center;
+}
+
+label,
+p {
+  margin: 0;
+}
+
+%stretch-panel {
+  flex-grow: 1;
+  margin: 10px;
+  padding: 20px;
+  border-radius: 20px;
+  background-color: white;
+}
+
+input[type="checkbox"] {
+  display: none;
+}
+
+input[type="checkbox"] + p {
```

Comment:
you can use classnames lib to avoid increasing specificity, just by setting class to element based on value, like:
```
{`test-class`: checkboxCheckedVariable}
```

Reply:
Also, let's use classes for styles, not the tag names

Reply:
That default styles in mate project) I'm dont write it)